cmake/Platform/Linux/Packaging/postinst.in

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+set -o errexit # exit on the first failure encountered
+
+{
+    if [[ ! -f "/usr/lib/x86_64-linux-gnu/libffi.so.6" ]]; then
+        sudo ln -s /usr/lib/x86_64-linux-gnu/libffi.so.7 /usr/lib/x86_64-linux-gnu/libffi.so.6
```

Comment:
Good catch on this, something that a user wouldn't normally see in Python after upgrading to libffi7

Reply:
This seems dangerous.
Instead of symlinking a new library as an older library, we should specify a dependency on [libffi6](https://packages.ubuntu.com/search?keywords=libffi6).

The problem with doing that though is that libffi6 is no longer available post Ubuntu 18.04 LTS

Reply:
yeah, I think this can be removed once we recompile python for linux on ubuntu 20